### ISSUE TITLE
Define microarray platform whitelist for Keytar Kurt

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio#228]
### Context

For Keytar Kurt planning, we'll need to define sets of experiments (e.g., lists of accession numbers) for our zebrafish (#227) and one million samples (#223) efforts. The first step in defining this list of experiments is identifying _supported platforms_ (and their identifiers in GEO, ArrayExpress, and SRA). We will use this ticket to define the supported **microarray** platforms for the Keytar Kurt milestone. (#145 is too broad for this purpose in my opinion.)

### Problem or idea

In a _broad sense_, we will have processors/support the following:

* **Affymetrix microarrays** (to be processed with `SCAN.UPC::SCANfast`)
* **Illumina microarrays** (to be processed with Illumina `SCAN` [being implemented as part of #212])
* RNA-seq short read (e.g., Illumina, Ion Torrent; see #159)

However, there some individual platforms (chips) that fall under these umbrella terms that we may not want to include in these first milestones (e.g., custom arrays).

### Solution or next step

I will define a list of supported microarray platforms. This depends on #220 (GEO-related) and the [list we _already have_ for ArrayExpress](https://github.com/AlexsLemonade/refinebio/issues/115#issuecomment-364265976). I will also perform #222 at the same time.

I will also note here that the following is the plan for Keytar Kurt:
* If we support the platform and there is raw data -> process from raw
* If we support the platform and there is no raw data available -> `NO OP` submitter processed data -> perform gene identifier mapping (#186)
* Else (the platform is unsupported) -> Do nothing/ignore/link out

(Please correct me if I'm wrong @kurtwheeler @Miserlou)

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by jaclyn-taroni** [comment]
I believe this has been addressed here: https://github.com/AlexsLemonade/refinebio/issues/222#issuecomment-389201432 and https://github.com/AlexsLemonade/refinebio/issues/222#issuecomment-389571978
